src/app/calculator/utilities/electricity-reduction/electricity-reduction.component.ts

```diff
@@ -166,6 +166,8 @@ export class ElectricityReductionComponent implements OnInit {
 
 
   getResults() {
+    this.electricityReductionService.baselineData = this.baselineData;
```

Comment:
Can move these to existing update___ methods. I guess double check the that the service wasn't already being updated.